lib/command/run-multiple.js

```diff
@@ -10,19 +10,20 @@ const runner = path.join(__dirname, '/../../bin/codecept');
 let config, childOpts = {};
 const copyOptions = ['steps', 'reporter', 'verbose', 'config', 'reporter-options', 'grep', 'fgrep'];
 
-// codeceptjs run:multiple smoke:chrome regression:firefox - will launch smoke suite in chrome and regression in firefox
-// codeceptjs run:multiple smoke:chrome regression - will launch smoke suite in chrome and regression in firefox and chrome
-// codeceptjs run:multiple all - will launch all suites
-// codeceptjs run:multiple smoke regression'
+// codeceptjs run:multiple smoke:chrome regression:firefox - will launch smoke suite in chrome and regression in
+// firefox
+// codeceptjs run:multiple smoke:chrome regression - will launch smoke suite in chrome and regression in firefox and
+// chrome codeceptjs run:multiple all - will launch all suites codeceptjs run:multiple smoke regression'
 
 module.exports = function (suites, options) {
   // registering options globally to use in config
   process.profile = options.profile;
   let configFile = options.config;
   let codecept;
-
   let testRoot = getTestRoot(configFile);
   config = getConfig(configFile);
+  let outputFolder = config.output;
+  let defaultWindowSize = searchValueInObj('windowSize', config.helpers)
```

Comment:
please remove defaultWindowSize. 
if no `windowSize` option in config, this option should be ignored.